gameShop/src/main/java/org/example/service/Singleton.java

```diff
@@ -0,0 +1,39 @@
+package org.example.service;
+
+import lombok.Getter;
+import org.example.AppData;
+import org.example.repository.GameRepositoryImpl;
+import org.example.repository.UserRepositoryImpl;
+import org.example.repository.dao.GameRepository;
+import org.example.repository.dao.UserRepository;
+
+import java.sql.Connection;
+
+@Getter
+public class Singleton {
```

Comment:
class name does not represent what class is responsible for, and you probably don't want this class to be a singleton